tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -572,9 +572,10 @@ async def test_cat_endpoints(wallet_rpc_environment: WalletRpcTestEnvironment):
     assert name == next(iter(DEFAULT_CATS.items()))[1]["name"]
 
     # TODO: Investigate why farming only one block here makes it flaky
-    await farm_transaction_block(full_node_api, wallet_node)
-    await farm_transaction_block(full_node_api, wallet_node)
+    for _ in range(5):
```

Comment:
does it not work with the previous 2 block setup ?
seems strange that it will change for the worst